presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -110,6 +110,7 @@
     private boolean forceSingleNodeOutput = true;
     private boolean pagesIndexEagerCompactionEnabled;
     private boolean distributedSort = true;
+    private boolean isEmptyJoinOptimization;
```

Comment:
Please follow naming convention in this file as well.

Reply:
Do you mean adding Optimize -> isOptimizeEmptyJoin or is it another naming convention?

Reply:
You can find in the variable definition above that we normally use `optimizeXXX` as variable names. `IsOptimizeXX` will be the getter method name.